test_communication/CMakeLists.txt

```diff
@@ -296,11 +474,23 @@ if(BUILD_TESTING)
     "test/test_requester.cpp")
   custom_executable(test_replier_cpp
     "test/test_replier.cpp")
+  # executables secure publisher / subscriber
+  custom_executable(test_secure_publisher_cpp
+    "test/test_secure_publisher.cpp")
+  custom_executable(test_secure_subscriber_cpp
+    "test/test_secure_subscriber.cpp")
 
   macro(targets)
     custom_test_c(test_messages_c
       "test/test_messages_c.cpp")
 
+    set(VALID_ROS_SECURE_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/test/test_security_files/")
+    # TODO(mikaelarguedas) extend this to connext once 5.2.4 released
```

Comment:
Confirmed 5.2.3 is the one you get after signing up for "Connext for Open Source Projects"